configuration/drugs/drugs.csv

```diff
@@ -284,4 +284,67 @@ a1ff1205-12ea-4511-b5c6-18eb16c4b6b2,,Multivit & Lysine Sirop Adulte,82cc193d-f7
+253c29ec-de6c-48b9-a5ab-2800db465fb3,,Multivit & Fer gttes (pr neonato),bcf73538-3f39-4429-a30d-8499609942cd,7c46524c-a14f-43b6-b4b7-792cee31d8d5,,
+ed53065e-0a23-4eb3-ae83-8fef9a106365,,Multivitamin & Fer,bcf73538-3f39-4429-a30d-8499609942cd,4d6a51aa-1c18-4757-bc40-9d9f567e31b8,,
+22abeb17-8d75-4e61-a027-fa0f19e1b83d,,Neomycin-Bacitracin zinc-polymixin B 5400ui/3.5g,a1b11bd9-f344-432a-877e-c12784c1acc1,44fc6700-a545-419e-a6b3-b882b6fc455c,5400ui/3.5g,
+af3f9ca4-37c5-404d-9c9b-b22b0584ed94,,Nifedepine 20mg,251a17f9-c229-4aee-9555-9bf7620b2a5a,fdbe907b-6abc-41dc-9599-2d84105e4063,20mg,
+cb2002f2-cc46-4b01-aa75-dbb9331e3cde,,Nystatin & Zinc Oxyde,a30dc1f2-a26b-4bc0-90a4-e7eb7feea473,44fc6700-a545-419e-a6b3-b882b6fc455c,,
+c5944a00-1e65-45e5-bff5-78fe39b06c5c,,Paracetamol 10mg/ml,9881091b-1802-4c62-8b80-a8fcb170b59f,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,10mg/ml,
+8dc9fcdc-b295-4c2f-ad4a-cc7328612cb9,,Pentoxyphillin 400mg,d644e9bd-7ae2-4ef3-b7b7-739816051213,fdbe907b-6abc-41dc-9599-2d84105e4063,400mg,
+44319262-7fab-404f-b0f6-fceeb050fe83,,Permetrine 1%,6315c226-01f6-4c59-9332-74f5347c3ef7,0689bb6e-9371-4a0f-ad1c-63c135b1714d,1%,
+85af3790-3429-43ba-8295-8821fbdcbb43,,Pfizer covid 19 vaccine 0.3ml/dose,0158e0b1-5fe4-4238-b06d-7fdede7b261a,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,0.3ml/dose,
+713ed568-43bd-4f6b-9f1d-12661fa80775,,Podophillin 0.15%,792871a0-4a52-4e5d-bdd2-4fd49d636227,44fc6700-a545-419e-a6b3-b882b6fc455c,0.15%,
+6248877a-34ec-4481-a76c-eca9e94f4e25,,Potassium chloride 20meq/15ml,a5547d6c-7e42-4a50-94e2-8e6e73aa6afe,16b302c7-715f-46cb-b328-c91afad4ccc8,20meq/15ml,
+255e4070-b718-4a6f-82f7-1d7ab89cc219,,Povidone iodine 10%,f7ae1d83-7687-45da-8327-8bf5532618e9,90a3f930-2eef-4a2e-8427-5359607c7b13,10%,
+87cd02b9-f08f-4bac-8618-f733cda51081,,Povidone iodine 9%,f7ae1d83-7687-45da-8327-8bf5532618e9,90a3f930-2eef-4a2e-8427-5359607c7b13,9%,
+64acfb5e-3345-4de6-9ba9-e5cff2108870,,Prednisolone 20mg,159bc783-b046-433f-93ef-9b33496f962e,fdbe907b-6abc-41dc-9599-2d84105e4063,20mg,
+426c1740-f9a2-4ca5-957d-dfbdcbd03db1,,Propranolol,062fc7f6-3d11-409a-96f7-a60cd8a71a36,0689bb6e-9371-4a0f-ad1c-63c135b1714d,,
+c96eb154-f5af-41bc-92a9-19e402b90e5b,,Salbutamol 4mg/5ml,9088359a-587f-4dbc-b13e-b3c38a07cd85,4d6a51aa-1c18-4757-bc40-9d9f567e31b8,4mg/5ml,
+4e690d52-3e36-45c8-a3d3-af6d777a5426,,Silver sulfadiazine 1%,89e4c545-85af-4d9c-9e8c-c8e54f815cf0,0689bb6e-9371-4a0f-ad1c-63c135b1714d,1%,
+3deec4fe-4a30-4ca5-8173-711f3a06ffac,,Simethicone 80mg,3fc5c46f-9c31-46ef-b96e-a12216f93a62,fdbe907b-6abc-41dc-9599-2d84105e4063,80mg,
+cf63a9b7-565e-4f44-9288-e120b4d3d589,,Spironolactone 50mg,d4c72168-24b0-4d22-ba60-848e7c146333,fdbe907b-6abc-41dc-9599-2d84105e4063,50mg,
+17844a70-1d2d-43a2-84e5-135ab455b6b9,,Vancomycin 500mg,41f47998-29ae-4f21-ad8c-32c50d0939f6,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,500mg,
+045d08d9-7205-497f-a2d8-1dc7f8b12826,,Vitamine A et D 50g,f72393c8-fda0-4107-9b4b-017888be72b2,0689bb6e-9371-4a0f-ad1c-63c135b1714d,50g,
```

Comment:
Lets use the existing concept drug [here](https://github.com/mekomsolutions/openmrs-config-haiti/pull/145/files#diff-c57c2d669cc45f0e2f0e74e8ddee7861933c6acc1cefe5a53b43bfb1ada816faR463)

Reply:
Done